pypyr/errors.py

```diff
@@ -16,9 +16,13 @@ class KeyInContextHasNoValueError(ContextError):
     """pypyr context[key] doesn't have a value."""
 
 
-class KeyNotInContextError(ContextError):
+class KeyNotInContextError(KeyError, ContextError):
```

Comment:
doesn't really matter, but keep alphabetical pls - `KeyNotInContextError(ContextError, KeyError)`

Reply:
Pls, look at my comment above if I change ordering the message string formatting will change, will became escaped and in double quotes, actually I didn't get why, I may guess that probably some of transformation happens on error init phase.

Reply:
may I just write? or is better to use super?
```
class KeyNotInContextError(ContextError, KeyError):
    """Key not found in the pypyr context."""    
    __str__ = Exception.__str__
```

Reply:
I don't know about _better_, re-binding `__str__` will achieve the same thing for sure, but given that we are in an inheritance/class derivation scenario, and ultimately KeyNotIInContextError inherits from Exception, it's probably more the expected OO thing to provide the `__str__` implementation with super.